raiden/tests/integration/fixtures/smartcontracts.py

```diff
@@ -98,17 +116,22 @@ def deploy_token_network_registry_and_return_address(
         settle_timeout_min,
         settle_timeout_max,
         contract_manager,
+        environment_type,
 ) -> typing.Address:
+    constructor_arguments = [
```

Comment:
Why not keep this a tuple as it was before?

Reply:
I cannot mutate the tuple... while i can for a list. Like #128 is where i append an additional item

Reply:
Ah I see apologies, did not see you were also mutating it